packages/@aws-cdk/aws-eks/lib/kubectl-handler/helm/__init__.py

```diff
@@ -109,8 +109,8 @@ def get_oci_cmd(repository, version):
             f"helm registry login --username AWS --password-stdin {registry}; helm pull {repository} --version {version} --untar"
             ]
     elif registry.startswith(public_ecr):
-        logger.info("Found AWS public repository, will use default region as deployment")
-        region = os.environ.get('AWS_REGION', 'us-east-1')
+        logger.info("Found AWS public repository, will use region 'us-east-1' as deployment")
+        region = 'us-east-1'
```

Comment:
I don't see a change to the unit tests for this; could you add a unit test for this change?

Reply:
I went through the previous commits for this lambda handler but didn't see any unit tests (unless I missed those) for these functions, so not sure what to add exactly here. What kind of unit test are you looking for? 

Note that the integration test https://github.com/aws/aws-cdk/blob/main/packages/%40aws-cdk/aws-eks/test/integ.eks-helm-asset.ts#L54 should already cover the behaviour change here as well, which I think makes more sense than a unit test in this case?

Reply:
Woops, this is a custom resource lambda. We have no unit tests for these, so disregard this.